src/commands/showReferrer.ts

```diff
@@ -0,0 +1,68 @@
+import * as vscode from 'vscode';
+import { execAsync } from '../support/execAsync';
+import { window } from 'vscode';
+import * as os from 'os';
+import * as path from 'path';
+import { access } from 'fs/promises'; //this import accepts one argument for calling access
+//TODO: Error Wrapping
+// check oras exist on user computer
+async function getOrasPath() {
+    const isWindows = os.platform() === 'win32'; //identifying if its a windows platform using os.platform, if it is then isWindows will be true
+    const homedir = os.homedir(); //getting the host computer home directory
+    const orasExecutable = isWindows ? 'oras.exe' : 'oras'; //switches use of executable files. if true use oras.exe and if false use oras
+    const dirOras = path.join(homedir, 'bin', 'oras', orasExecutable); //returns the path to oras
+
+    try {
+        await access(dirOras); // Check if the file exists, this will throw an error if the file doesn't exist
+        return dirOras; // If the file exists, return the dirOras path
+    } catch (err) {
+         // Throw an error with the message
```

Comment:
Nit. This is comment is improperly spaced. Consider running the format command for vscode or explore the usage of linters.